src/StatsTools/ratio_of_means.jl

```diff
@@ -84,8 +95,8 @@ means is propagated using `MonteCarloMeasurements`. The results are reported
 as a [`RatioBlockingResult`](@ref).
 
 Robust estimates for the ratio
-are obtained from [`median(r)`](@ref) and confidence intervals from [`quantile()`](@ref),
-e.g. `quantile(r, [0.025, 0.975])` for the 95% confidence interval.
+are obtained from [`pmedian(r)`](@ref) and confidence intervals from [`pquantile()`](@ref),
```

Comment:
Hmm, the `(@ref)` links will be broken unless we document the `p*` functions ourselves.

Reply:
Agreed, I'll remove the `@ref`.